src/java.base/share/classes/jdk/internal/classfile/impl/DirectCodeBuilder.java

```diff
@@ -416,6 +439,11 @@ public void writeTo(BufWriterImpl b) {
             b.writeU2(buf.size() / 4);
             b.writeBytes(buf);
         }
+
+        @Override
+        public Utf8Entry attributeName() {
+            return NAME;
```

Comment:
Maybe `buf.constantPool().utf8Entry(NAME_LINE_NUMBER_TABLE)`?

Reply:
Good catch, I missed there is buf field initialized.